app/code/community/Vindi/Subscription/Helper/Validator.php

```diff
@@ -25,17 +25,16 @@ public function validateChargeWebhook($data)
 		$charge = $data['charge'];
 		$order = $this->orderHandler->getOrderFromVindi($charge['bill']['id']);
 		
-		if (! $order) {
-			$this->logWebhook('Pedido não encontrado.', 4);
-			return false;
+		if (! $order || true === $order) {
+			return $order;
```

Comment:
como retorna `$order` se for `! $order`?

Reply:
O _$order_ pode ser:
- **false**
- **null**
- **true**
- **Mage_Sales_Order**

Nesse caso, a validação desse retorno é feita no [controller](https://github.com/vindi/vindi-magento/blob/0e1869b44de9f77eee46af790737fb99444559f3/app/code/community/Vindi/Subscription/controllers/WebhookController.php#L27)

Reply:
Então não precisa do `if`, certo? É só retornar `true`